gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -82,6 +97,8 @@
   // The size of this object generation, in bytes.
   private final long objectSize;
 
+  private final MetricsPublisher metricsPublisher;
```

Comment:
What is the gap that we are trying to close with custom gRPC metrics? Did we try enable [default gRPC metrics](https://opencensus.io/guides/grpc/java/#1) and see if they are sufficient for our needs?